app/src/main/java/com/willowtree/vocable/utils/FaceTrackingPermissions.kt

```diff
@@ -23,4 +43,82 @@ class FaceTrackingPermissions(private val sharedPreferences: IVocableSharedPrefe
         sharedPreferences.setHeadTrackingEnabled(false)
         permissionState.tryEmit(Disabled)
     }
+
+    private val permissionLauncher: ActivityResultLauncher<String> =
+        activity.registerForActivityResult(ActivityResultContracts.RequestPermission()) { isGranted: Boolean ->
+            if (isGranted) {
+                enableFaceTracking()
+            } else {
+                disableFaceTracking()
+                permissionSettingsDialog.show()
+            }
+        }
```

Comment:
all the new stuff in this file is just moved wholesale from CameraPermissionsHelper. The only intended changes are to directly use `enableFaceTracking()` or `disableFaceTracking()` instead of calling them via callbacks

Reply:
Since enable/disable are now called internally, do they still need to be part of the public interface here?

This then feeds into how this class is being tested, which I think is going to inform whether or not your mocking below is sufficient. My gut says it's not, since you're now unable to test the various branching points at line 49